src/kernel/src/dev/sbl_srv.rs

```diff
@@ -19,3 +30,32 @@ impl DeviceDriver for SblSrv {
         todo!()
     }
 }
+
+pub struct SblSrvManager {
+    sbl: Arc<CharacterDevice>,
+}
+
+impl SblSrvManager {
+    pub fn new() -> Result<Arc<Self>, SblSrvInitError> {
+        let sbl = make_dev(
+            SblSrv {},
+            DriverFlags::from_bits_retain(0x80000004),
```

Comment:
Where did you get this value from?

Reply:
From the branch. But it seems it was wrong